edgelet/iotedge/src/main.rs

```diff
@@ -40,8 +40,9 @@ fn main() {
 
 #[allow(clippy::too_many_lines)]
 fn run() -> Result<(), Error> {
-    let (default_mgmt_uri, default_config_path, default_container_engine_config_path) =
+    let (default_secret_uri, default_mgmt_uri, default_config_path, default_container_engine_config_path) =
         if cfg!(windows) {
+            panic!("NO WINDOWS SUPPORT");
```

Comment:
it should handle no windows support inside secret store api, but not panic here

Reply:
This was due to a mistake on my part where I edited `iotedge` rather than `iotedged`. The change can be ignored, and I will remove it.